### ISSUE TITLE
Code for passing the grasp pose to the real robot arm

### ISSUE THREAD

**Issue by zzzzzzj0** [luyh20/VL-Grasp#4]
Hello, thank you for your excellent work. I am trying to program the robotic arm to grip according to the output pose. I have converted between different coordinate systems in the code, but the result is still not correct.This problem really bothers me,  I was wondering if the author could provide the source code for passing the grasp pose to the real robot arm. Thank you very much.

**Comment by luyh20** [comment]
Notice that the predicted rot parameter from the model is the rotation matrix from the grasp pose to the original axis of the coordinate system in visualization. And the visualization coordinate maybe not match with your real gripper TCP coordinate. Maybe you need to set a ratation matrix from gripper TCP coordinate to visualization coordniate.

**Comment by zzzzzzj0** [comment]
Might I ask if the "coordinate system in visualization" you mentioned refers to the camera coordinate system? I have performed hand-eye calibration and obtained the transformation matrix from the camera coordinate system to the robot arm base coordinate system. I then applied this transformation to the rotation matrix outputted by the model, resulting in the rotation matrix in the robot arm base coordinate system. However, while the grasp position is correct, the grasp orientation is not.
